docs/make.jl

```diff
@@ -2,70 +2,30 @@ Base.HOME_PROJECT[] = abspath(Base.HOME_PROJECT[]) # JuliaLang/julia/pull/28625
 
 using CLIMA, Documenter, Literate
 
-# TODO: Add generated examples back
-# include("generate.jl")
+examples_dir = joinpath(@__DIR__, "..", "examples")      # julia src files
+generated_dir = joinpath(@__DIR__, "generated") # generated files directory
+mkpath(generated_dir)
+generate_examples = true
 
-GENERATED_BL_EXAMPLES = [
-    joinpath("examples", "DGmethods_old", "generated", f)
-    for
-    f in ("ex_001_periodic_advection.md", "ex_002_solid_body_rotation.md")
-]
-GENERATED_BL_EXAMPLES = filter!(x -> isfile(x), GENERATED_BL_EXAMPLES)
-
-# Literate examples created in their own literate_example directory for now
```

Comment:
These lines allow us to stick all the literate files into a single directory, which I thought was what we wanted? Right now, all the literate examples go in the `literate_examples` subdirectory and are generated automatically with this.

Reply:
Ok, great! I didn't know that `examples` was supposed to be purely literate files, which was why we made the change before.